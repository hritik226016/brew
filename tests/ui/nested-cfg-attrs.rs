#[cfg_attr(all(), cfg_attr(all(), cfg(FALSE)))] //~ NOTE the item is gated here
fn f() {} //~ NOTE found an item that was configured out

fn main() { f() } //~ ERROR cannot find function `f` in this scope
//~^ NOTE not found in this scope
