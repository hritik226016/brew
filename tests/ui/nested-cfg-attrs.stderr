error[E0425]: cannot find function `f` in this scope
  --> $DIR/nested-cfg-attrs.rs:4:13
   |
LL | fn main() { f() }
   |             ^ not found in this scope
   |
note: found an item that was configured out
  --> $DIR/nested-cfg-attrs.rs:2:4
   |
LL | fn f() {}
   |    ^
note: the item is gated here
  --> $DIR/nested-cfg-attrs.rs:1:35
   |
LL | #[cfg_attr(all(), cfg_attr(all(), cfg(FALSE)))]
   |                                   ^^^^^^^^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0425`.
